.github/workflows/build.yml

```diff
@@ -7,10 +7,11 @@ on:
   pull_request:
 jobs:
   build:
-    name: Build
+    name: Build & Install
```

Comment:
should you rename the file to reflect changes?

Reply:
it is `build.yml`

Reply:
I'm not 100% certain yet. I was kind of planning ot see what @prologic has to say about that.

Reply:
Rename it :) Thats the convention I'm using.

Reply:
Can we also:

```sh
mkdir .github/dockerfiles
mv .github/Dockerfile.build_test .github/dockerfiles/
```

I'd prefer it if we keep logical directory structures for our own sanity ;)